.github/ISSUE_TEMPLATE/3-plugin-compatiblity.md

```diff
@@ -0,0 +1,27 @@
+---
+name: "💥 Plugin compatibility"
+about: Create a plugin compatibility issue report
+---
+
+### Your checklist for this issue
```

Comment:
Should this link to the compatibility page or have a short sentence on it here?

Reply:
The contribution page mentions it but we can mention it specifically here instead if you'd prefer 😄